src/stravalib/model.py

```diff
@@ -448,8 +436,6 @@ class Athlete(strava_model.DetailedAthlete):
 
     # Field overrides from superclass for type extensions:
     clubs: Optional[list[SummaryClub]] = None
-    bikes: Optional[list[SummaryGear]] = None
-    shoes: Optional[list[SummaryGear]] = None
 
     # Undocumented attributes:
     athlete_type: Optional[Literal["cyclist", "runner"]] = None
```

Comment:
@jsamoocha when i look at the model object, i see a LOT of undocumented attributes. 

```python
# Undocumented attributes:
    athlete_type: Optional[Literal["cyclist", "runner"]] = None
    friend: Optional[str] = None
    follower: Optional[str] = None
    approve_followers: Optional[bool] = None
    badge_type_id: Optional[int] = None
    mutual_friend_count: Optional[int] = None
    date_preference: Optional[str] = None
    email: Optional[str] = None
    super_user: Optional[bool] = None
    email_language: Optional[str] = None
    max_heartrate: Optional[float] = None
    username: Optional[str] = None
    description: Optional[str] = None
    instagram_username: Optional[str] = None
    offer_in_app_payment: Optional[bool] = None
    global_privacy: Optional[bool] = None
    receive_newsletter: Optional[bool] = None
    email_kom_lost: Optional[bool] = None
    dateofbirth: Optional[date] = None
    facebook_sharing_enabled: Optional[bool] = None
    profile_original: Optional[str] = None
    premium_expiration_date: Optional[int] = None
    email_send_follower_notices: Optional[bool] = None
    plan: Optional[str] = None
    agreed_to_terms: Optional[str] = None
    follower_request_count: Optional[int] = None
    email_facebook_twitter_friend_joins: Optional[bool] = None
    receive_kudos_emails: Optional[bool] = None
    receive_follower_feed_emails: Optional[bool] = None
    receive_comment_emails: Optional[bool] = None
    sample_race_distance: Optional[int] = None
    sample_race_time: Optional[int] = None
    membership: Optional[str] = None
    admin: Optional[bool] = None
    owner: Optional[bool] = None
    subscription_permissions: Optional[list[bool]] = None
```


Note - this could be related to my account type - i don't know... but my return in postman has fewer attributes. 
Should we consider removing some of these undocumented attrs? OR do you get a different return than i do?
OR - am i just missing something which is also totally possible! 

<img width="723" alt="Screenshot 2024-06-15 at 7 35 34 PM" src="https://github.com/stravalib/stravalib/assets/7649194/07cae251-0aaf-4725-bbd4-38122121aaca">

this is the endpoint i'm testing `https://www.strava.com/api/v3/athlete`

Reply:
TBH, I never checked which of these undocumented attributes still existed in Strava responses. Most of them have been there for ages. I made a new issue for this, see #528.